elements/nuxeo-document-create-popup/nuxeo-document-create-popup.js

```diff
@@ -62,8 +70,8 @@ Polymer({
       }
 
       #holder {
-        height: 80vh;
-        width: 65vw;
+        height: calc(var(--nuxeo-document-create-popup-height, 80vh) - 48px);
```

Comment:
Since we are already using `var(--nuxeo-document-create-popup-height, 80vh)` on this elements' parent, we should be able to just do:
```suggestion
        height: calc(100% - 48px);
```